src/main/org/deidentifier/arx/criteria/EDDifferentialPrivacy.java

```diff
@@ -169,18 +178,12 @@ public int getRequirements(){
     @Override
     public void initialize(DataManager manager, ARXConfiguration config){
         
-        if (config != null) {
-            // Because this is the only privacy model that dynamically
-            // constructs a subset, we need to call this method twice.
-            // This is the second call and can be ignored.
-            return;
-        }
-        
         // Set beta and k if required
         if (beta < 0) {
+            double epsilonAnon = epsilon - (isDataDependent() ? config.getDPSearchBudget() : 0d);

```

Comment:
Seems to not be a reliable calculation, although errors are obviously very small.